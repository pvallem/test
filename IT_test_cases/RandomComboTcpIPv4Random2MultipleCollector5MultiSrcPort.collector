bidir_flow_info {
  key {
    src_address: "\001\002\003\000"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000010
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\026\3562}"
    dst_address: "4(tq"
    src_port: 59542
    dst_port: 12178
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 10
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\216m\215Z"
    dst_address: "\265\030h\244"
    src_port: 4921
    dst_port: 40894
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000040
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "Gm\332j"
    dst_address: "z\367\025\033"
    src_port: 55881
    dst_port: 46434
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 0
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\002\003\001"
    dst_address: "\001\002\003d"
    src_port: 12346
    dst_port: 12345
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
