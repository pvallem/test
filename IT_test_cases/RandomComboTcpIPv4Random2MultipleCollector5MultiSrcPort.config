test_name: "RandomComboTcpIPv4Random2MultipleCollector5MultiSrcPort"
sensor_config {
  sensor_binary: "./test_data/host-sensor"
  config_file: "/home/tetter/tetration/test_framework/src/integration_test/IT_test_cases/RandomComboTcpIPv4Random2MultipleCollector5MultiSrcPorteth0.sw_sensor_config"
}
collector_config {
  collector_binary: "./test_data/Collector"
  collector_config_file: "./test_data/collector_proto.config"
}
sensor_output_protobuf_file: "/home/tetter/tetration/test_framework/src/integration_test/IT_test_cases/RandomComboTcpIPv4Random2MultipleCollector5MultiSrcPort.golden"
collector_output_protobuf_file: "/home/tetter/tetration/test_framework/src/integration_test/IT_test_cases/RandomComboTcpIPv4Random2MultipleCollector5MultiSrcPort.collector"
test_time_out_sec: 4
last_stage: collector
run_locally: true
stub_collector_count: 1
sensor_type: software_sensor
asic_configs {
  asic_binary: "/mnt/tet-11/workspace/ASICUnitTest/tahoe/model/tetration/asic_sensor"
  config_file: "/home/tetter/tetration/test_framework/src/integration_test/IT_test_cases/RandomComboTcpIPv4Random2MultipleCollector5MultiSrcPort.sensor_config"
  packet_file: "/home/tetter/tetration/test_framework/src/integration_test/IT_test_cases/RandomComboTcpIPv4Random2MultipleCollector5MultiSrcPort.packet"
  output_file: "/home/tetter/tetration/test_framework/src/integration_test/IT_test_cases/RandomComboTcpIPv4Random2MultipleCollector5MultiSrcPort.out"
  backup_path: "./test_data/asic_test_data"
  events_file: "/home/tetter/tetration/test_framework/src/integration_test/IT_test_cases/RandomComboTcpIPv4Random2MultipleCollector5MultiSrcPort.events"
  rtl_dump_file: "/home/tetter/tetration/test_framework/src/integration_test/IT_test_cases/RandomComboTcpIPv4Random2MultipleCollector5MultiSrcPort.rtldump"
  rtl_config_dump_file: "/home/tetter/tetration/test_framework/src/integration_test/IT_test_cases/RandomComboTcpIPv4Random2MultipleCollector5MultiSrcPort.rtl_config"
}
