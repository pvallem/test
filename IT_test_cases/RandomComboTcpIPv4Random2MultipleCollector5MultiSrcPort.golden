flow_info {
  key {
    src_address: "z\367\025\033"
    dst_address: "Gm\332j"
    src_port: 46434
    dst_port: 55881
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 0
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\001\002\003\000"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV4
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000010
}
flow_info {
  key {
    src_address: "\026\3562}"
    dst_address: "4(tq"
    src_port: 59542
    dst_port: 12178
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  start_time: 1388563200000020
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 10
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\001\002\003\001"
    dst_address: "\001\002\003d"
    src_port: 12346
    dst_port: 12345
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV4
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000030
}
flow_info {
  key {
    src_address: "\216m\215Z"
    dst_address: "\265\030h\244"
    src_port: 4921
    dst_port: 40894
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV4
  }
  start_time: 1388563200000040
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000040
}
